tendrl/gluster_integration/message/gluster_native_message_handler.py

```diff
@@ -0,0 +1,95 @@
+from flask import Flask
+from flask import request
+import gevent.event
+import gevent.greenlet
+
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils import service as s
+from tendrl.gluster_integration.message import callback as cb
+
+
+app = Flask(__name__)
+
+
+class GlusterNativeMessageHandler(gevent.greenlet.Greenlet):
+    def __init__(self):
+        super(GlusterNativeMessageHandler, self).__init__()
+        self.path = "/listen"
+        self.host = "0.0.0.0"
+        self.port = 9000
+        self.callback = cb.Callback()
+
+        @app.route(self.path, methods=["POST"])
+        def events_listener():
+            gluster_event = request.json
+            if gluster_event:
+                callback_function_name = gluster_event["event"].lower()
+                try:
+                    function = getattr(self.callback, callback_function_name)
+                except AttributeError:
+                    # tendrl does not handle this perticular event hence ignore
```

Comment:
Should we log debug at least for this as user would never come to know.

Reply:
I observed lots of events being raised by glustereventsd. I feel its unnecessary noise, even logging it in debug mode as these are events that tendrl does not care about.
Open to inputs

Reply:
I am ok. Lets hear from others.

Reply:
The events which are raised by glustereventsd - they are also logged by the Gluster system, are they not? Asking for my own clarification to ensure that events outside of the current focus of the Tendrl system are still being logged for audit/review.

Reply:
As per my discussion with @aravindavk. Yes, the events will be logged locally unless user explicitly disables it using `disable-events-log`.